clients/java/signalr/build.gradle

```diff
@@ -32,10 +32,16 @@ spotless {
         importOrder 'java', 'javax', 'org', 'com', 'com.diffplug', ''	// A sequence of package names
 
         replace      'Not enough space after if', 'if(', 'if ('
+        replace      'Not enough space after else', 'else {', 'else {'
```

Comment:
Typo, this currently doesn't do anything

Reply:
Good eye